server/channels/app/file.go

```diff
@@ -1363,7 +1376,7 @@ func (a *App) CopyFileInfos(rctx request.CTX, userID string, fileIDs []string) (
 
 // This function zip's up all the files in fileDatas array and then saves it to the directory specified with the specified zip file name
 // Ensure the zip file name ends with a .zip
-func (a *App) CreateZipFileAndAddFiles(fileBackend filestore.FileBackend, fileDatas []model.FileData, zipFileName, directory string) error {
+func (a *App) CreateZipFileAndAddFiles(rctx request.CTX, fileBackend filestore.FileBackend, fileDatas []model.FileData, zipFileName, directory string) error {
```

Comment:
You need to update a couple of files based on the change. Please check if the code compiles locally and fix any errors.